main_test.go

```diff
@@ -36,6 +36,11 @@ func Test_getVars(t *testing.T) {
 			config: "A=B\n# comment\nC=D",
 			want:   []string{"A=B", "C=D"},
 		},
+		{
+			name:   "one var with equals in value",
+			config: `A="abc123="`,
```

Comment:
Since config-shim doesn't support quotes around values, this test case should probably be changed to `A=abc123=`

Reply:
Scratch that. I forgot I added rudimentary support for quotes.

Reply:
I was (we were?) surprised when that test passed before we had implemented the fix, so something about how that works was/is still unclear to me.